internal/runners/packages/searchView.go

```diff
@@ -0,0 +1,304 @@
+				break
+			}
+			versions = append(versions, styleCyan.Render(v))
+		}
+		if len(versions) > 0 {
+			doc.WriteString(formatRow(styleBold.Render(keyVersions), strings.Join(versions, ", "), maxKeyLength, v.width))
+		}
+
+		if len(pkg.Vulnerabilities) > 0 {
+			var (
+				critical = pkg.Vulnerabilities["Critical"]
+				high     = pkg.Vulnerabilities["High"]
+				medium   = pkg.Vulnerabilities["Medium"]
+				low      = pkg.Vulnerabilities["Low"]
+			)
+
+			vunlSummary := []string{}
+			if critical > 0 {
+				vunlSummary = append(vunlSummary, styleRed.Render(locale.Tl("search_critical", "{{.V0}} Critical", strconv.Itoa(critical))))
+			}
+			if high > 0 {
+				vunlSummary = append(vunlSummary, styleOrange.Render(locale.Tl("search_high", "{{.V0}} High", strconv.Itoa(high))))
+			}
+			if medium > 0 {
+				vunlSummary = append(vunlSummary, styleYellow.Render(locale.Tl("search_medium", "{{.V0}} Medium", strconv.Itoa(medium))))
+			}
+			if low > 0 {
+				vunlSummary = append(vunlSummary, styleMagenta.Render(locale.Tl("search_low", "{{.V0}} Low", strconv.Itoa(low))))
+			}
+
+			if len(vunlSummary) > 0 {
+				doc.WriteString(formatRow(styleBold.Render(keyVulns), strings.Join(vunlSummary, ", "), maxKeyLength, v.width))
+			}
+		}
+
+		doc.WriteString("\n")
+	}
+	return doc.String()
+}
+
+func (v *view) initialRemaining() {
+	// Interate over all of the content and calculate the index.
+	// Each time we encounter a new package name, we determine how many
+	// remaining packages there are and store that in the index.
+	lines := strings.Split(v.content, "\n")
+	index := 1
+	v.index = make(map[string]int)
+	for _, l := range lines {
+		if strings.Contains(l, "Name") {
+			v.index[l] = index
+			index++
+		}
+	}
+
+	visibleContent := strings.Split(v.viewport.View(), "\n")
+	for _, l := range visibleContent {
+		if strings.Contains(l, "Name") {
+			v.remaining = len(v.searchResults.Results) - v.index[l]
+		}
+	}
+
+	if v.remaining < 0 {
+		v.remaining = 0
+	}
+}
+
+func (v *view) footerView() string {
+	var footerText string
+	if v.remaining != 0 {
+		footerText += locale.Tl("search_more_matches", "... {{.V0}} more matches, use arrow and page keys to scroll. Press Q to quit.", strconv.Itoa(v.remaining))
+	}
+	footerText += fmt.Sprintf("\n\n%s'%s'", styleBold.Render(locale.Tl("search_more_info", "For more info run")), styleActionable.Render(locale.Tl("search_more_info_command", " state info <name>")))
+	return lipgloss.NewStyle().Render(footerText)
+}
+
+func (v *view) scroll(direction string, amount int) {
+	if direction == "up" {
+		// LineUp returns the new lines. In order to update the remaining count
+		// we need to iterate over the visible lines and find the remaining count
+		// for the last visible package name.
+		_ = v.viewport.LineUp(amount)
+		lines := strings.Split(v.viewport.View(), "\n")
+		for _, l := range lines {
+			if strings.Contains(l, "Name") {
+				v.remaining = len(v.searchResults.packageNames) - v.index[l]
+			}
```

Comment:
This is very hacky and prone to bugs. I'd rather we just show the percentage if the alternative is comparing strings.

Reply:
While I like how this works, I'm also weary about this approach. I've updated it to use the `ScrollPercent()` function. The localization is my best guess so I expect you will want to update that.